guides/source/asset_pipeline.md

```diff
@@ -184,9 +184,8 @@ the file `scaffolds.css` (or `scaffolds.scss` if `sass-rails` is in the
 `Gemfile`.)
 
 For example, if you generate a `ProjectsController`, Rails will also add a new
-file at `app/assets/javascripts/projects.coffee` and another at
-`app/assets/stylesheets/projects.scss`. By default these files will be ready
-to use by your application immediately using the `require_tree` directive. See
+file at `app/assets/stylesheets/projects.scss`. By default these files will be
```

Comment:
misspelled ? `these files` -> `this file` ?

Reply:
No, I missed updating it. Earlier there were two files, now only one. Will take care of it another round of review. Thanks 👍